github_jira_sync_app/settings.yaml

```diff
@@ -8,3 +8,4 @@ settings:
   jira_project_key:
   label_mapping:
   status_mapping:
+  summary: ""
```

Comment:
we either make everything in the YAML as empty string, or just leave it empty

what is the YAML standard way of doing it ?

Reply:
we can let this empty, fixed